packages/atomic/cypress/integration/breadbrumb-manager.cypress.ts

```diff
@@ -66,7 +66,7 @@ describe('Breadcrumb Manager Test Suites', () => {
       });
     });
 
-    it('should remove the breadcrumb when it is clicked', () => {
+    it.skip('should remove the breadcrumb when it is clicked', () => {
```

Comment:
There is hope Lucille's upgrade to the latest cypress will help resolve this 🙏

Reply:
Yes, upgrading cypress fix the stuff. @ThibodeauJF already merged my branch, so maybe you can pull master into your branch.